examples/run_deephaven.py

```diff
@@ -0,0 +1,16 @@
+
+# This script runs the Deephaven server with the specified port and JVM arguments.
+# The server will run until the script is interrupted.
+#
+# To connect to the Deephaven IDE, navigate to https://localhost:10000
+# The login password is: DeephavenRocks!
+
+import os
+from time import sleep
+from deephaven_server import Server
+
+_server = Server(port=10000, jvm_args=['-Xmx4g','-Dauthentication.psk=DeephavenRocks!','-Dstorage.path=' + os.path.expanduser('~/.deephaven')])
```

Comment:
Consider using click to specify the JVM args, such as more memory or a different password.

Reply:
I'm going back and forth on this one.

1. I can see using click because it is awesome and the script would be better.
2. I can see not using click because this is intended to be a very simple example, and click might confuse/muddy the simplicity.

Reply:
It can be given default parameters that are the same as what's given here, but I'd envision it used something like this:

```sh
python3 run_deephaven.py --port=9999 --password=SomeOtherPasswordHere
```

I won't let this block an approval though. Let me know when you decide whether or not to use it, and I can look again.

Reply:
I completely see that path, and I like it.  I'm just concerned that if we do that a user won't write their own run scripts.  I don't have a lean.  Let's see what @alexpeters1208 thinks.

Reply:
Is there a way to get the new CLI to work with the `os.path.expanduser` argument? I suspect probably not. If so, I would vote to axe this script and just recommend the new CLI tooling. Otherwise, I would probably vote to use click for a similar startup experience.